pronto-jscs.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency('pronto', '~> 0.5.0')
+  spec.add_runtime_dependency('pronto', '~> 0.5')
```

Comment:
It's a little different...

`~> 0.5.0` will allow 0.5.0, 0.5.1, 0.5.2 ... 0.5.9999 but that's it.
`~> 0.5` will allow 0.5.0, 0.5.1, 0.5.2 ... 0.6.0, 0.6.1, 0.6.2 ... 0.9999.9999

Right now, for example, the latest version of `pronto` is 0.6.0, but `pronto-jscs` cannot use it. (Which is lucky, in this case, as `pronto` has a breaking method signature change in a minor version update (0.5.0 - 0.6.0), which shouldn't happen).